docs/platform.rst

```diff
@@ -1,43 +1,23 @@
 Aiven platform
 ==================
 
-What is the Aiven platform?
-----------------------------
-
 Aiven provides managed open source data technologies on all major clouds. Through Aiven, developers can do what they do best: create applications. Meanwhile, Aiven does what it does best; manage cloud data infrastructure. 
 
-Aiven platform consists of Aiven tools, and the following Aiven products:
-Apache Kafka®,
-Apache Cassandra®,
-PostgreSQL®,
-Apache Flink®,
-ClickHouse®,
-OpenSearch®,
-MySQL,
-Redis®*,
-InfluxDB®,
-Grafana®,
-and M3 - all available in more than 80 regions around the world on AWS, GCP, Microsoft Azure, DigitalOcean, and UpCloud cloud platforms.
-
-Learn more about the Aiven platform
-------------------------------------
-
-.. grid:: 1 2 2 2
-
-    .. grid-item-card::
-        :shadow: md
-        :margin: 2 2 0 0
-
-        📚 :doc:`Concepts </docs/platform/concepts>`
+The Aiven platform consists of the :doc:`Aiven Console </docs/tools/aiven-console>` and :doc:`tools </docs/tools>`, and the Aiven services:
```

Comment:
Consider removing the 1st "and" and using a comma.
`Aiven Console, tools, and services`